SQL/basic-sql-queries.txt

```diff
@@ -134,3 +134,37 @@ SELECT TOP 2 * FROM <name of table>;
 
 /* To update a specific column value in the row */
 UPDATE <Table Name> SET <Clm Name> = <'Alter value'> where <Clm name> <any arthmatic operators> <value>;
+
+6.
```

Comment:
remove the line 138.
the number '6.' is not needed, pls remove all such numberings below